### PR TITLE
CLN: Use fixture dtype_backend in nullable tests

### PR DESCRIPTION
**Pull request by phofl**
- [ ] closes #xxxx (Replace xxxx with the GitHub issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

File: pandas/tests/tools/test_to_numeric.py [hunk 1]
```diff
@@ -912,7 +912,6 @@ def test_to_numeric_use_nullable_dtypes_already_nullable(dtype):
 @pytest.mark.parametrize(
     "use_nullable_dtypes, dtype", [(True, "Float64"), (False, "float64")]
 )
-@pytest.mark.parametrize("dtype_backend", ["pandas", "pyarrow"])
 def test_to_numeric_use_nullable_dtypes_error(
```
Comment (mroeschke) [inline pandas/tests/tools/test_to_numeric.py:915]: I think the `pytest.importorskip("pyarrow")` can be removed within the test now

File: pandas/tests/io/test_parquet.py [hunk 2]
```diff
@@ -640,6 +641,7 @@ def test_use_nullable_dtypes(self, engine, request):
             expected = expected.drop("c", axis=1)
         tm.assert_frame_equal(result2, expected)
 
+    @pytest.mark.skipif(pa_version_under6p0, reason="minimum pyarrow not installed")
```
Comment (mroeschke) [inline pandas/tests/io/test_parquet.py:644]: Doesn't `engine` already do a similar `skipif`?

File: pandas/tests/io/test_parquet.py [hunk 3]
```diff
@@ -591,6 +591,7 @@ def test_write_column_index_nonstring(self, pa):
         msg = r"parquet must have string column names"
         self.check_error_on_write(df, engine, ValueError, msg)
 
+    @pytest.mark.skipif(pa_version_under6p0, reason="minimum pyarrow not installed")
```
Comment (mroeschke) [inline pandas/tests/io/test_parquet.py:594]: Similarly here?

File: pandas/tests/io/test_parquet.py (hunk 2 above)
Comment (phofl) [reply pandas/tests/io/test_parquet.py:644]: I thought so as well, but it was failing locally without pyarrow

Comment (phofl) [reply pandas/tests/io/test_parquet.py:644]: Ah sorry, actually it's only skipped for pyarrow engine but it is imported for both engines, e.g. the test will fail if fast parquet engine is used

File: pandas/tests/io/test_parquet.py (hunk 3 above)
Comment (phofl) [reply pandas/tests/io/test_parquet.py:594]: Same as above

File: pandas/tests/tools/test_to_numeric.py (hunk 1 above)
Comment (phofl) [reply pandas/tests/tools/test_to_numeric.py:915]: Ah good point, thx

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @phofl

**Pull request merged**